### PR TITLE
disabe autopay for some cases

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_debt_notebook/models.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def init_debt_journal(self):
                                                 'debt_dummy_product_id': self.env.ref('pos_debt_notebook.product_pay_debt').id,
                                                 'debt_limit': default_debt_limit,
                                                 'pos_cash_out': True,
-                                                'credits_autopay': True,
+                                                'credits_autopay': False,
```
Comment (yelizariev) [inline pos_debt_notebook/models.py]: I think we should keep it True, i.e. activate feature in new installation

**Review by yelizariev — CHANGES_REQUESTED** [review]
Add copyrights and license to all files, please

File: pos_debt_notebook/static/src/js/pos.js [hunk 2]
```diff
@@ -727,7 +734,7 @@ odoo.define('pos_debt_notebook.pos', function (require) {
                 return;
             }
             this._super();
-            if (!order.get_paymentlines().length){
+            if (!order.get_paymentlines().length || order.has_credit_product()){
```
Comment (yelizariev) [inline pos_debt_notebook/static/src/js/pos.js]: ``render_paymentlines`` probably need to be updated. It looks strange, that it makes update only when there are no lines

**Review by yelizariev — APPROVED** [review]
Code is ok

**Review by ilmir-k — APPROVED** [review]

**Pull request merged**